files/en-us/web/api/document_object_model/locating_dom_elements_using_selectors/index.html

```diff
@@ -35,12 +45,11 @@ <h2 id="Selectors">Selectors</h2>
 
 <p>After executing the above code, <code>el</code> contains the first element in the document whose ID is one of <code>main</code>, <code>basic</code>, or <code>exclamation</code>.</p>
 
-<p>You may use any CSS selectors with the <code>querySelector()</code> and <code>querySelectorAll()</code> methods.</p>
-
 <h2 id="See_also">See also</h2>
 
 <ul>
- <li><a href="https://www.w3.org/TR/selectors-api/">Selectors API</a></li>
+ <li><a href="https://www.w3.org/TR/selectors-3/">Selectors Level 3 (W3C Recommendation)</a></li>
+ <li>{{cssxref("CSS_Selectors", "CSS Selectors")}}</li>
```

Comment:
```suggestion
 <li><a href="/en-US/docs/Web/CSS/CSS_Selectors">CSS Selectors</a></li>
```
…because the `cssxref` macro otherwise causes it to get rendered as `css selectors` (lowercase, monospace).

In general, anyway, it’s always OK to not use the macros but instead put in a normal hyperlink.

In fact, these days the devs working on the MDN infrastructure/tooling strongly prefer that we use normal hyperlinks rather than macros (in part I guess because of needing to deal with inquiries/confusion about the macros doing stuff that some people (like me) don’t expect them to do.

(I can imagine why the `cssxref` does this automatic lowercase+monospace things, but I guess I don’t reference the CSS docs often enough to have run into before myself. Or at least I don’t remember…)